Renderer/Block/AtomicBlockRenderer.php

```diff
@@ -89,10 +89,17 @@ public function getName()
      * @param array $entities
      *
      * @return null|DraftEntity
+     *
+     * @throws DraftjsException
      */
     protected function getEntity(ContentBlock $contentBlock, array $entities)
     {
         $characterList = $contentBlock->getCharacterList();
+
+        if (!isset($characterList[0])) {
+            throw new DraftjsException(sprintf('Block "%s" with key "%s" is invalid. At least 1 character is required.', self::TYPE, $contentBlock->getKey()));
+        }
+
         $characterMetadata = $characterList[0];
         $index = $characterMetadata->getEntityIndex();
 
```

Comment:
In fact, keep errors silent is an perfectly assuming choice we did to avoid the full fail of rendering that could be due to a wrong part of content state. I admit it may be more relevant to throw clear exceptions for an easier debugging but I think we should find a more smarter way to keep working of rendering at least partially if necessary for production environment.